env.dev.pg.base

```diff
@@ -28,28 +15,28 @@ WDK_OAUTH_CLIENT_SECRET=wdkoauthsecret
 SERVICE_OAUTH_CLIENT_ID=veupathdbService
 SERVICE_OAUTH_CLIENT_SECRET=serviceoauthsecret
 
-OAUTH_DB_PLATFORM=Oracle
+OAUTH_DB_PLATFORM=PostgreSQL
 
 # app db (WDK)
 APP_DB_LOGIN=your_dblogin
```

Comment:
Maybe we should take these out of here since they are in the sample?  I originally put everything in base to sort of document the entire set of env vars, then letting sample/custom override, but now thinking that is a mistake and we should have values in here that make sense to hard code but anything that is likely to be overridden should not be here and only in sample.